sdk/agrifood/agrifood-farming-rest/package.json

```diff
@@ -77,30 +77,30 @@
   "sideEffects": false,
   "autoPublish": false,
   "dependencies": {
-    "@azure-rest/core-client": "^1.0.0",
-    "@azure/core-auth": "^1.3.0",
-    "@azure/core-lro": "^3.1.0",
-    "@azure/logger": "^1.0.0",
-    "tslib": "^2.2.0",
-    "@azure/abort-controller": "^2.1.2"
+    "@azure-rest/core-client": "workspace:*",
```

Comment:
Since codegen haven't adopted these changes, so I want to understand if our ci still use the old version pattern, will the build and test fail?

Reply:
@MaryGao it should continue to work as long as the package is available via npm, meaning it will not work with those internal to us such as @azure/dev-tool which are not published.  We will need to fix up at least the internal modules so that we can continue to work.

Reply:
Got it and let us double verify that.